bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "mastery-exercise-single-page-app-MattKraatz",
+  "homepage": "https://github.com/nss-day-cohort-15/mastery-exercise-single-page-app-MattKraatz",
+  "authors": [
+    "Matt Kraatz <matt.kraatz@gmail.com>"
+  ],
+  "description": "",
+  "main": "",
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ]
+}
```

Comment:
Don't forget to add the `--save` when you install a dependency so that anyone who pulls your code down can just `bower install` all the required dependencies.